tests/unit/test__pandas_helpers.py

```diff
@@ -29,13 +29,9 @@
     import pandas.testing
 except ImportError:  # pragma: NO COVER
     pandas = None
-try:
```

Comment:
I notice a test failure:

```
________________ test_dataframe_to_parquet_dict_sequence_schema ________________

module_under_test = <module 'google.cloud.bigquery._pandas_helpers' from '/tmpfs/src/github/python-bigquery/google/cloud/bigquery/_pandas_helpers.py'>

    @pytest.mark.skipif(isinstance(pyarrow, mock.Mock), reason="Requires `pyarrow`")
    def test_dataframe_to_parquet_dict_sequence_schema(module_under_test):
        dict_schema = [
            {"name": "field01", "type": "STRING", "mode": "REQUIRED"},
            {"name": "field02", "type": "BOOL", "mode": "NULLABLE"},
        ]

>       dataframe = pandas.DataFrame(
            {"field01": [u"hello", u"world"], "field02": [True, False]}
        )
E       AttributeError: 'NoneType' object has no attribute 'DataFrame'

tests/unit/test__pandas_helpers.py:1252: AttributeError
```

Can we use `importorskip` here for pandas?

Reply:
Done.